.github/workflows/publish-packages.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish packages
```

Comment:
This is just .github/workflows/v2-merged-staging.yml without the staging publish (which no longer worked).